changelog/deprecation_asm_stc.dd

```diff
@@ -0,0 +1,6 @@
+The deprecation period of unannotated `asm` blocks has been ended.
+
+See the $(LINK2 $(ROOT_DIR)deprecate.html#asm, Deprecated Features) for more information.
+
+Starting with this release, using unannotated `asm` blocks will result in a
```

Comment:
They won't result in a compiler error per se, they'll be assumed to be the opposite of `pure` `@safe` `nothrow` or `@nogc.

Reply:
Changed the wording